wheels_power_enabled: 8
motion_enabled: 9

hw_ok: 10
wdog_ok: 11
laser_ok: 12
edm_ok: 13
selector_ok: 14
hw_ok_estop: 15
vel_ok: 16

op_mode_auto: 17
op_mode_manual: 18
op_mode_maintenance: 19
# op_mode_emergency: 20

switch_laser_mute_on: 21
switch_laser_mute_off: 22

lsr1_safezone_free: 24
lsr2_safezone_free: 25
lsr1_warningzone1_free: 26
lsr2_warningzone1_free: 27
lsr1_warningzone2_free: 28
lsr2_warningzone2_free: 29

lsr_mode_standard_input: 32
lsr_mode_docking_input: 33
lsr_mode_standby_input: 34
lsr_mode_cart_input: 35
lsr_mode_error: 39

platform_moving: 36
mode_changed: 37
reset_required: 38

# OUTPUT

emergency_stop_sw: 0

buzzer_on_pulse: 3
buzzer_on_intermittent: 4

buzzer_mode_2: 17
buzzer_mode_3: 18

lsr_mode_standard_output: 26
lsr_mode_docking_output: 27
lsr_mode_standby_output: 28
lsr_mode_cart_output: 29